rpcserver.go

```diff
@@ -1079,15 +1078,27 @@ func (r *rpcServer) WalletBalance(ctx context.Context,
 		}
 	}
 
-	balance, err := r.server.cc.wallet.ConfirmedBalance(1, in.WitnessOnly)
+	// Get total balance, from txs that have >= 0 confirmations
+	TotalBal, err := r.server.cc.wallet.ConfirmedBalance(0, in.WitnessOnly)
 	if err != nil {
 		return nil, err
 	}
 
-	rpcsLog.Debugf("[walletbalance] balance=%v", balance)
+	// Get confirmed balance, from txs that have >= 1 confirmations
+	ConfirmedBal, err := r.server.cc.wallet.ConfirmedBalance(1, in.WitnessOnly)
```

Comment:
nit: make `ConfirmedBal ` => `confirmedBal` because this is an intermediate variable.